pact/coin-contract/coin-sig.pact

```diff
@@ -22,20 +22,28 @@
 
   (defun transfer-and-create:string (sender:string receiver:string receiver-guard:guard amount:decimal)
     @doc "Transfer between accounts SENDER and RECEIVER on the same chain.    \
-    \This fails if the SENDER account does not exist. If the RECEIVER account \
-    \does not exist, it is created and associated with GUARD."
+    \This fails if both accounts do not exist. Create-on-transfer can be      \
```

Comment:
Not sure I like this change.  The new wording `both accounts do` seems less clear to me than what was there before.  Also, the last sentence is weird because this *is* the `transfer-and-create` function.

Reply:
This is the wording from the new docs you wrote in your last PR to `coin.pact`. I'm just making sure the documentation is flush. Do you want me to amend the comments here and in `coin.pact`?

Reply:
Ah I added it to the wrong transfer function. My bad. Regarding the wording, this is what you wrote for the `transfer` function in your recent changes to `coin.pact`. Do you want me amend it?